src/vs/workbench/contrib/terminal/browser/terminalLinkHandler.ts

```diff
@@ -274,19 +275,27 @@ export class TerminalLinkHandler {
 		return isMacintosh ? event.metaKey : event.ctrlKey;
 	}
 
-	private _getLinkHoverString(): string {
+	private _getLinkHoverString(uri: string): IMarkdownString {
 		const editorConf = this._configurationService.getValue<{ multiCursorModifier: 'ctrlCmd' | 'alt' }>('editor');
+
+		let label = '';
 		if (editorConf.multiCursorModifier === 'ctrlCmd') {
 			if (isMacintosh) {
-				return nls.localize('terminalLinkHandler.followLinkAlt.mac', "Option + click to follow link");
+				label = nls.localize('terminalLinkHandler.followLinkAlt.mac', "Option + click");
 			} else {
-				return nls.localize('terminalLinkHandler.followLinkAlt', "Alt + click to follow link");
+				label = nls.localize('terminalLinkHandler.followLinkAlt', "Alt + click");
 			}
 		}
 		if (isMacintosh) {
-			return nls.localize('terminalLinkHandler.followLinkCmd', "Cmd + click to follow link");
+			label = nls.localize('terminalLinkHandler.followLinkCmd', "Cmd + click");
 		}
-		return nls.localize('terminalLinkHandler.followLinkCtrl', "Ctrl + click to follow link");
+		label = nls.localize('terminalLinkHandler.followLinkCtrl', "Ctrl + click");
```

Comment:
After this change it looks like some else-branches are required. Without return, the wrong label is used.